concepticondata/conceptlists/Smeltzer-1995-300.tsv

```diff
@@ -0,0 +1,301 @@
+Smeltzer-1995-300-183	183	corps	1480	BODY	body
+Smeltzer-1995-300-184	184	plaie	1109	WOUND	wound
+Smeltzer-1995-300-185	185	veine	1924	VEIN	vein
+Smeltzer-1995-300-186	186	larmes	1651	TEAR (OF EYE)	tear
+Smeltzer-1995-300-187	187	excrément	675	FAECES (EXCREMENT)	excrement
+Smeltzer-1995-300-188	188	huile	1484	OIL (ORGANIC SUBSTANCE)	oil
+Smeltzer-1995-300-189	189	mère	1216	MOTHER	mother
+Smeltzer-1995-300-190	190	frère aîné	1759	OLDER BROTHER	older brother
+Smeltzer-1995-300-191	191	petite soeur	1761	YOUNGER SISTER	younger sister
+Smeltzer-1995-300-192	192	ami	1325	FRIEND	friend
+Smeltzer-1995-300-193	193	oncle (côté matèrnelle.)	3220	MATERNAL UNCLE	uncle (mother's side)
+Smeltzer-1995-300-194	194	invité	1782	GUEST	guest
+Smeltzer-1995-300-195	195	chef	301	CHIEFTAIN	chieftain
+Smeltzer-1995-300-196	196	âme	811	SOUL	soul
+Smeltzer-1995-300-197	197	lion	1386	LION	lion
+Smeltzer-1995-300-198	198	rat	1490	RAT	rat
+Smeltzer-1995-300-199	199	chauve-souris	1793	BAT	bat
+Smeltzer-1995-300-200	200	cheval	615	HORSE	horse
+Smeltzer-1995-300-201	201	pintade	456	GUINEA FOWL	guinea fowl
+Smeltzer-1995-300-202	202	lézard	632	LIZARD	lizard
+Smeltzer-1995-300-203	203	crocodile	1857	CROCODILE	crocodile
+Smeltzer-1995-300-204	204	ver	1219	WORM	worm
+Smeltzer-1995-300-205	205	abeille	665	BEE	bee
+Smeltzer-1995-300-206	206	fourmis	587	ANT	ant
+Smeltzer-1995-300-207	207	araignée	843	SPIDER	spider
+Smeltzer-1995-300-208	208	scorpion	1538	SCORPION	scorpion
+Smeltzer-1995-300-209	209	bâton	1295	STICK	stick
+Smeltzer-1995-300-210	210	épine	124	THORN	thorn
+Smeltzer-1995-300-211	211	fleur	239	FLOWER	flower
+Smeltzer-1995-300-212	212	fruit	1507	FRUIT	fruit
+Smeltzer-1995-300-213	213	mil	931	MILLET	millet
+Smeltzer-1995-300-214	214	maïs	506	MAIZE	maize
+Smeltzer-1995-300-215	215	banane	868	BANANA	banana
+Smeltzer-1995-300-216	216	arachide	2364	PEANUT	peanut
+Smeltzer-1995-300-217	217	herbe	606	GRASS	grass
+Smeltzer-1995-300-218	218	“la brousse“	1683	BUSH	bush
+Smeltzer-1995-300-219	219	champ	212	FIELD	field
+Smeltzer-1995-300-220	220	pays	1300	COUNTRY	country
+Smeltzer-1995-300-221	221	porte	1567	DOOR	door
+Smeltzer-1995-300-222	222	chose	738	THING	thing
+Smeltzer-1995-300-223	223	pagne	1965	LOINCLOTH	loincloth
+Smeltzer-1995-300-224	224	calebasse	411	GOURD	gourd
+Smeltzer-1995-300-225	225	trou	1667	HOLE	hole
+Smeltzer-1995-300-226	226	mortier	224	MORTAR	mortar
+Smeltzer-1995-300-227	227	pilon	225	PESTLE	pestle
+Smeltzer-1995-300-228	228	hache	677	AXE	axe
+Smeltzer-1995-300-229	229	flèche	977	ARROW	arrow
+Smeltzer-1995-300-230	230	charbon	2658	COAL	coal
+Smeltzer-1995-300-231	231	matin	1339	MORNING	morning
+Smeltzer-1995-300-232	232	foudre	1654	LIGHTNING	lightning
+Smeltzer-1995-300-233	233	houe	284	HOE	hoe
+Smeltzer-1995-300-234	234	argile	1399	CLAY	clay
+Smeltzer-1995-300-235	235	vingt	1710	TWENTY	twenty
+Smeltzer-1995-300-236	236	cent	1634	HUNDRED	hundred
+Smeltzer-1995-300-237	237	profond	1593	DEEP	deep
+Smeltzer-1995-300-238	238	grand	711	TALL	tall, large
```

Comment:
This also seems to be an ambiguous case. I found some list that mapped grand to BIG (e.g., Meillet-1921-16-13 | grand [french])

Reply:
This one is extremely ambiguous. As far as I understood from Aymeric's explanation, its proximity to 'gros' points to a certain kind of 'big' in the vertical dimension which is tall in this case. On the other hand, 'gros' refers to a big size more in the horizontal sense.